buttercup.el

```diff
@@ -800,8 +1239,10 @@ current directory."
     (dolist (file command-line-args-left)
       (with-current-buffer (find-file-noselect file)
         (goto-char (point-min))
-        (while (re-search-forward "```lisp\n\\(\\(?:.\\|\n\\)*?\\)```"
-                                  nil t)
+        (while (let ((case-fold-search t))
```

Comment:
You can put this `let` around the `while` for minor performance wins.

Reply:
I'll make this change.

Reply:
Ok, this is addressed in e9eada9.